get_databricks_cluster_eventlog.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Default directory to save results to
+results_dir="$PWD/databricks_cluster_eventlogs"
+mkdir -p $results_dir
+
+function print_usage() {
+  cat <<-EOD
```

Comment:
probably because I pirated it from your log-parser bash script! 😛 

Totally agree it makes the script look nicer.

Reply:
The only downside is someone maintaining the script might overlook the tab characters.

Reply:
Oh woah, that must have come from the copy-paste from your script, because my vscode automatically replaces tabs with spaces. Good eye! Can you identify \t from github somehow?

Reply:
I don't think so - must be VSCode